src/hooks/use-companies-selection-menu-item.js

```diff
@@ -0,0 +1,19 @@
+import { useEffect, useState } from 'react';
+
+const useCompaniesSelectionMenuItem = (companies) => {
+  const [selectedCompany, setSelectedCompany] = useState('');
+
+  useEffect(() => {
+    if (companies?.length) {
+      setSelectedCompany(companies[0].id);
```

Comment:
Since the name of the state is `selectedCompany` I think you could store the whole company object, and then you could use only the `id`, or use more data if needed.

```suggestion
      setSelectedCompany(companies[0]);
```

Alternatively, you could change the name of the state to something like `selectedCompanyId`, then it would make more sense that you store only the company's ID